cmd/influxd/run.go

```diff
@@ -187,6 +188,31 @@ func Run(config *Config, join, version string) (*messaging.Broker, *influxdb.Ser
 			}
 		}
 
+		// Spin up any OpenTSDB servers
+		if config.OpenTSDB.Enabled {
```

Comment:
You must do this inside the block above, since data ingestion only takes place on nodes which have a "server" object.